kernel/api/pom.xml

```diff
@@ -98,5 +98,20 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.12.1</version>
```

Comment:
these  versions should be controlled in master pom

Reply:
Agreed. Good spot.